did/did.go

```diff
@@ -110,6 +109,27 @@ func ParseDID(input string) (*DID, error) {
 	return did, nil
 }
 
+// Must accepts a function like Parse and returns the value without error or panics otherwise.
+func Must(fn func(string) (*DID, error), input string) DID {
```

Comment:
Should this be exported?

Reply:
It's very specific (given the signature) and all parse functions have already been covered. I'll make it private